### ISSUE TITLE
what is the scope of the Solid spec?

### ISSUE THREAD

**Issue by Mitzi-Laszlo** [solid/specification#29]
On the 3rd August 2019 @namedgraph_twitter asked: 

"what is the scope of the Solid spec?" on the [solid/solid-spec gitter channel](https://gitter.im/solid/solid-spec). 

There was a conversation answering that questions and it would be good to record it in an issue for future reference.

**Comment by Mitzi-Laszlo** [comment]
Here is a possible answer to the question posed:

"The aim of the Solid specification is to provide the necessary information for data storage providers and application providers to build in such a way that their users can switch between app providers and data storage providers and take the data generated along." 

If you agree with this please respond to this with a thumbs up. 

If you have an improved version of the answer to the question please post it in this thread.

**Comment by RubenVerborgh** [comment]
I think the above answer conflates two things: 1) what is (the scope of) Solid 2) what is (the scope of) the Solid specification.

I don't think that 1 should be answered in the scope of the specification.

Here's an alternative take:

"The Solid specification is a set of technical documents that detail the requirements for apps, clients, servers, and intermediaries to be compatible with Solid."

**Comment by csarven** [comment]
"to be compatible with Solid" kind of leaves it hanging. Too meta or self-referential for my taste. "What's 'Solid' again?"

I generally agree that the question/answer can be split to indicate the scope of 1 and 2 but a bit of an overlap or cross referencing is fine. After all, the Solid Ecosystem is rooted in what's Solid's vision/scope any way. So, in that regard, I don't find Mitzi's description to be inadequate. It can be improved. And, that sort of a brief entry text (eg. in Abstract or Instruction of Solid Ecosystem) would be useful IMO. It sets the stage.. At the very least, it can refer to a more complete description of what "Solid" is - which is being worked on in parallel any way.

FWIW, just to exemplify, LDN's https://www.w3.org/TR/ldn/#introduction says this:

>Data on the Web should not be locked in to particular systems or be only readable by the applications which created it. Users should be free to switch between applications and share data between them.

Of course that's not specific to LDN but it frames a Solid-y perspective.

I don't think that The Solid Ecosystem should go into an essay mode or anything like that - because it is primarily about the technicalities - but briefly giving context and pointers to relevant material can help eg. https://www.w3.org/TR/ldn/#social-web-working-group . I think that sort of thing should happen in all Solid specs so that the reader can navigate vertically in addition to horizontal spec reading.

Different ways to slice this.. and both of your descriptions (or at least the approach) are equally useful for different reasons for The Solid Ecosystem doc.

**Comment by RubenVerborgh** [comment]
> "to be compatible with Solid" kind of leaves it hanging. Too meta or self-referential for my taste. "What's 'Solid' again?"

Not intended to be self-referential, but rather to point to a definition of Solid (which would also be needed).

**Comment by Mitzi-Laszlo** [comment]
Any updates from the [Explaining the Vision Panel](https://github.com/solid/process/blob/master/panels.md#explaining-the-vision-panel) on [What is Solid](https://github.com/solid/roadmap/blob/master/what-is-solid.md) proposal?

**Comment by Mitzi-Laszlo** [comment]
Bingo! https://github.com/solid/Explaining-the-Vision-Panel/blob/master/Tim-describing-Solid/20160708_Decentralized-Web-Summit.md#20160708_decentralized-web-summit_3222

**Comment by csarven** [comment]
Coming back to this issue with fresh eyes.. it may just be a duplicate of #9 and what's intended:

* Publish a Use Cases and Requirements document.
* Have the Solid Ecosystem spec provide an easy-to-understand narrative to describe situations that are applicable to the Solid specifications - a general summary of the UCR document (perhaps in the Introduction or nearby). Link to UCR for details.

How does that sound?
